iphone/Classes/TiAppiOSProxy.m

```diff
@@ -224,7 +224,7 @@ - (void)didReceiveApplicationShortcutNotification:(NSNotification *)info
   RELEASE_TO_NIL(event);
 }
 
-#ifdef USE_TI_APPIOSUSERNOTIFICATIONCENTER
+#if defined(USE_TI_APPIOSUSERNOTIFICATIONCENTER) && !TARGET_OS_MACCATALYST
```

Comment:
This should not be necessary, since you wrap it at the beginning of the file already